src/server/editorServices.ts

```diff
@@ -70,7 +80,12 @@ namespace ts.server {
         readonly dts: number;
     }
 
-    export type ProjectServiceEvent = ProjectsUpdatedInBackgroundEvent | ConfigFileDiagEvent | ProjectLanguageServiceStateEvent | ProjectInfoTelemetryEvent;
+    export interface OpenFileStats {

```

Comment:
i thought we will just send an event on every new file `.js` we see in openfile, and will have a flag whether it has checkjs or not. we can do the counting at the server side. no need for counting here.

Reply:
Done